app/views/shared/_header.html.erb

```diff
@@ -1,4 +1,4 @@
-<header role="eui-banner">
+<header class="mmt-header" role="eui-banner">
```

Comment:
We have a `header` tag in manage_metadata and manage_cmr layouts as well -- do we need this change there as well?